TI_mergesort.c

```diff
@@ -0,0 +1,72 @@
+#include <stdio.h>
+
+void merge(int arr[], int start, int mid, int end) {
+
+  int len1 = mid - start + 1;
+  int len2 = end - mid;
+
+  int leftArr[len1];
+  int rightArr[len2];
+
+  for (int i = 0; i < len1; i++)

```

Comment:
hier befüllt man die Arrays in der Funktion mit den Werten der erstellten Array in der main (falls ich da was falsch verstanden habe bitte ein comment da lassen)